kubernetes/launch-job.sh

```diff
@@ -94,7 +94,6 @@ if [ -n "${USE_PREDICTOR}" ]; then
       --gpu 0 1 \
       --name go-training-"$1"-predictor
 else
-  PREDICTOR_DIR=""
```

Comment:
`PREDICTOR_FLAG` will be undefined in the `else` branch. It'll work, but seems bad practice. In fact I'd be in favour of adding the `-u` flag to the shebang line at the top which would treat this as an error.